nickspaargaren/no-google
# Add dns check to domains

This PR adds a pipeline that lists the domains that don't have an A record. 

ELI5: Google domains that don't exist anymore should be listed or automatically removed in by Github Action. With this change we can remove unused domains to make the list more compact.

### How to test
1. Run `make` to see the new `make dnscheck` command.
2. Run the new `make dnscheck` command.
3. Make sure all domains without a response get automaticcly removed.
